cmd/context/use.go

```diff
@@ -124,18 +124,21 @@ func (c *ContextCommand) Run(ctx context.Context, ctxOptions *ContextOptions) er
 
 func getContext(ctx context.Context, ctxOptions *ContextOptions) (string, error) {
 	ctxs := getContextsSelection(ctxOptions)
-	oktetoContext, isOkteto, err := utils.AskForOptionsOkteto(ctx, ctxs, "A context defines the default cluster/namespace for any Okteto CLI command.\nSelect the context you want to use:", "Context")
+	initialPosition := getInitialPosition(ctxs)
+	selector := utils.NewOktetoSelector("A context defines the default cluster/namespace for any Okteto CLI command.\nSelect the context you want to use:", "Context")
+	oktetoContext, err := selector.AskForOptionsOkteto(ctxs, initialPosition)
 	if err != nil {
 		return "", err
 	}
-	ctxOptions.IsOkteto = isOkteto
 
 	if isCreateNewContextOption(oktetoContext) {
 		oktetoContext, err = askForOktetoURL()
 		if err != nil {
 			return "", err
 		}
 		ctxOptions.IsOkteto = true
+	} else {
+		ctxOptions.IsOkteto = selectedIsOkteto(oktetoContext)
```

Comment:
I'm confused with this change within the function. `ctxOptions` is being used to get the list of contexts to display and then, we modify its value based on what the user has selected. In general, this is a smell and error prone. Function parameters should be immutable and ideally this function should return all the information needed so the caller can change the `ctxOptions` based on the response.

Anyways, it is out of the scope of this PR as this is how it is now